packages/mobile/src/exchange/saga.ts

```diff
@@ -338,6 +339,68 @@ function* createStandbyTx(
   return txId
 }
 
+function navigateToExchangeScreenIfReviewing() {
+  navigateIf(
+    (nav) => nav?.getCurrentRoute()?.name === Screens.WithdrawCeloReviewScreen,
+    Screens.ExchangeHomeScreen
+  )
+}
+
+function* withdrawCelo(action: WithdrawCeloAction) {
+  let txId: string | null = null
+  try {
+    const { recipientAddress, amount } = action
+    const account: string = yield call(getConnectedUnlockedAccount)
+
+    txId = generateStandbyTransactionId(account)
+    yield put(
+      addStandbyTransaction({
+        id: txId,
+        type: TokenTransactionType.Sent,
+        comment: '',
+        status: TransactionStatus.Pending,
+        value: amount.toString(),
+        symbol: CURRENCY_ENUM.GOLD,
+        timestamp: Math.floor(Date.now() / 1000),
+        address: recipientAddress,
+      })
+    )
+
+    setTimeout(() => {
+      navigateToExchangeScreenIfReviewing()
+    }, 3000)
```

Comment:
Why 3 seconds and why do we call `navigateToExchangeScreenIfReviewing` again after some timeout?

Reply:
I talked to Corey about this, after this screen the `ExchangeHomeScreen` shows an item as 'Confirming' until the transaction is confirmed, we hoped to give it a few seconds to confirm before going back in hopes that in many cases the user doesn't see the 'Confirming' text, but on second thought I don't think it makes much sense because it won't cover all cases and it adds unnecessary complexity to the code imo, I'll talk to him to see if it's ok to remove this.